README.md

```diff
@@ -1,5 +1,5 @@
 # consul-k8s-prometheus-grafana-hashicups-demoapp
-This repo contains application and dashboard definitions for the Consul Layer 7 observability with Kubernetes guide located at learn.hashicorp.com
+This repo contains application and dashboard definitions for the <a target="_blank" href="https://learn.hashicorp.com/tutorials/consul/kubernetes-layer7-observability">Consul Layer 7 observability with Kubernetes guide located at learn.hashicorp.com</a>
```

Comment:
let's use markdown for links!

```suggestion
This repo contains application and dashboard definitions for the [Consul Layer 7 observability with Kubernetes guide](https://learn.hashicorp.com/tutorials/consul/kubernetes-layer7-observability) located at learn.hashicorp.com.
```

Reply:
OK. The difference is that GitHub translates to links that take people away from a page.
But using it is more consistent. I'll do it your way from here one.